src/components/AddBook.js

```diff
@@ -1,23 +1,23 @@
 import React, { useState } from 'react';
-import { useDispatch } from 'react-redux';
 import { v4 as uuidv4 } from 'uuid';
-import { addBook } from '../redux/books/books';
-
-const AddBook = () => {
-  const dispatch = useDispatch();
+import PropTypes from 'prop-types';
 
+const AddBook = ({ handleSubmit }) => {
+  const bookCategories = ['Fiction', 'Classics', 'Mystery', 'Contemporary'];
+  const rand = Math.floor(Math.random() * bookCategories.length);
+  const category = bookCategories[rand];
   const [newBook, setNewBook] = useState(
     {
-      id: '',
+      item_id: uuidv4(),
       title: '',
       author: '',
+      category: '',
     },
   );
 
   const handleSubmitBook = (e) => {
     e.preventDefault();
-    setNewBook({ ...newBook, id: uuidv4 });
-    dispatch(addBook(newBook));
+    handleSubmit({ ...newBook, category });
   };
 
   return (
```

Comment:
Optional. 
Consider clearing the input field after a book has been submitted. This will enhance the user experience.
![image](https://user-images.githubusercontent.com/33268927/191783896-85003608-d03c-47f5-ab77-fac27412673b.png)

Reply:
Done